nifi-registry-framework/src/main/java/org/apache/nifi/registry/provider/hook/LoggingEventHookProvider.java

```diff
@@ -36,6 +36,10 @@ public void onConfigured(final ProviderConfigurationContext configurationContext
 
     @Override
     public void handle(final Event event) throws EventHookException {
+
+        // Purposely leaving off "handleEvent" logic as its assumed that everything should be logged.
+        //handleEvent(event)
```

Comment:
but that contradicts the documentation of the shared property... again I think we need to discuss what the intention is and then implement the code to match